index.html

```diff
@@ -8,9 +8,10 @@
   <body>
     <div class="nav-bar">
       <ul>
-        <li>Home</li>
+        <li><a href="index.html">home</a></li>
         <li>Stores</li>
-        <li>Contact Us</li>
+        <li><a href="mailto:Salmon.cookies@buisness.com">Contact Us</a></li>
+        <li><a href="sales.html">Pat's Store Data</a></li>
       </ul>
     </div>
     <headder>
```

Comment:
probably an extra d in there it appears to still be creating a block element though...